src/components/Pokemon/PokemonList/PokemonList.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect,useState} from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+
+const StyledDiv=styled.div`
+    text-transform:uppercase;
+    color:#00ccff;
+    &:hover{
+       color:#eed603;
+     }
+`;
+
+const PokemonList=(props)=>{
+    
+    const setImageState=useState([]);
```

Comment:
In general, it is easier to use useState with destructuration like this [imageState, setImageState] where imageState is the actual state and setImageState is used for setting the state